scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -1415,11 +1411,6 @@ static __always_inline void p2dq_dispatch_impl(s32 cpu, struct task_struct *prev
 		return;
 	}
 
-	if (p2dq_config.nr_dsqs_per_llc > MAX_DSQS_PER_LLC) {
-		scx_bpf_error("can't happen");
-		return;
-	}
```

Comment:
I don't see much point removing this given it'll be jitted out, but if the verifier's happy I'm happy.